cmd/src/update.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+
+	"github.com/tj/go-update"
+	"github.com/tj/go-update/progress"
+	"github.com/tj/go-update/stores/github"
+)
+
+const (
+	// PrivateFileMode grants owner to read/write a file.
+	PrivateFileMode = 0600
```

Comment:
Does this need to be exported?

```suggestion
	privateFileMode = 0600
```